ecosystem/platform/server/app/controllers/it2_profiles_controller.rb

```diff
@@ -45,11 +45,7 @@ def create
     if @it2_profile.save
       log @it2_profile, 'created'
 
-      if Flipper.enabled?(:node_health_checker)
-        @it2_profile.enqueue_nhc_job(true)
-      else
-        validate_node(v, do_location: true)
-      end
+      @it2_profile.enqueue_nhc_job(true)
```

Comment:
Delete the old validate_node / validate_node_nhc methods if they're no longer used.

Reply:
I would suggest creating a separate PR that makes this change and also deletes the old NodeVerifier code.